### DISCUSSION TITLE
Connect to vaultwarden with the new Bitwarden Beta app

### DISCUSSION THREAD

**Discussion by ln-12** [dani-garcia/vaultwarden#4655] [Q&A] [answered]
A few hours ago, the beta for the completely rewritten Bitwarden iOS and Android apps were published: https://fosstodon.org/@bitwarden/112639925158165590

Sadly, I cannot manage to log in to my existing vaultwarden instance using this beta (on Android). The old app works perfectly fine. I don't see any debug menu in the app to get some logs but maybe someone here has an idea where to look.

Most importantly, we shoudl figure out if this is an issue in the Bitwarden app or Vaultwarden.

**Comment by BlackDex** [comment, marked as answer]
See #4386
